public/js/app.js

```diff
@@ -50251,9 +50251,9 @@ __webpack_require__.r(__webpack_exports__);
 /*! no static exports found */
 /***/ (function(module, exports, __webpack_require__) {
 
-__webpack_require__(/*! /Users/mattstauffer/Sites/gistlog/resources/js/app.js */"./resources/js/app.js");
-__webpack_require__(/*! /Users/mattstauffer/Sites/gistlog/resources/less/landing.less */"./resources/less/landing.less");
-module.exports = __webpack_require__(/*! /Users/mattstauffer/Sites/gistlog/resources/less/app.less */"./resources/less/app.less");
+__webpack_require__(/*! /Users/ant/Sites/tighten/gistlog/resources/js/app.js */"./resources/js/app.js");
```

Comment:
Hey @mattstauffer , this is the result of `npm run dev` but I'm not sure it belongs in this PR. Let me know if you'd like it dropped 😄

Reply:
Naw that’s fine